drivers/pinmux/stm32/pinmux_stm32h7.h

```diff
@@ -13,33 +13,83 @@
  */
 
 /* Port A */
+#define STM32H7_PINMUX_FUNC_PA0_PWM2_CH1 \
+	(STM32_PINMUX_ALT_FUNC_1 | STM32_PUSHPULL_PULLUP)
+#define STM32H7_PINMUX_FUNC_PA0_PWM5_CH1 \
+	(STM32_PINMUX_ALT_FUNC_2 | STM32_PUSHPULL_PULLUP)
 #define STM32H7_PINMUX_FUNC_PA0_USART2_CTS    \
 	(STM32_PINMUX_ALT_FUNC_7 | STM32_PUSHPULL_PULLUP)
 #define STM32H7_PINMUX_FUNC_PA0_UART4_TX    \
 	(STM32_PINMUX_ALT_FUNC_8 | STM32_PUSHPULL_PULLUP)
 
+#define STM32H7_PINMUX_FUNC_PA1_PWM2_CH2 \
+	(STM32_PINMUX_ALT_FUNC_1 | STM32_PUSHPULL_PULLUP)
+#define STM32H7_PINMUX_FUNC_PA1_PWM5_CH2 \
+	(STM32_PINMUX_ALT_FUNC_2 | STM32_PUSHPULL_PULLUP)
+#define STM32H7_PINMUX_FUNC_PA1_PWM9_CH1N \
+	(STM32_PINMUX_ALT_FUNC_4 | STM32_PUSHPULL_PULLUP)
 #define STM32H7_PINMUX_FUNC_PA1_USART2_RTS    \
 	(STM32_PINMUX_ALT_FUNC_7 | STM32_PUSHPULL_PULLUP)
 #define STM32H7_PINMUX_FUNC_PA1_UART4_RX    \
 	(STM32_PINMUX_ALT_FUNC_8 | STM32_PUSHPULL_NOPULL)
 
+#define STM32H7_PINMUX_FUNC_PA2_PWM2_CH3 \
+	(STM32_PINMUX_ALT_FUNC_1 | STM32_PUSHPULL_PULLUP)
+#define STM32H7_PINMUX_FUNC_PA2_PWM5_CH3 \
+	(STM32_PINMUX_ALT_FUNC_2 | STM32_PUSHPULL_PULLUP)
+#define STM32H7_PINMUX_FUNC_PA2_PWM9_CH1 \
+	(STM32_PINMUX_ALT_FUNC_4 | STM32_PUSHPULL_PULLUP)
```

Comment:
For stm32H742/743, the PA1 pin with Alternate Function 4 is assigned to TIM15_CH1 (or PWM15_CH1) and not PWM9_CH1

Reply:
@FRASTM Same as above